cloudsync_onedrive.py

```diff
@@ -1177,9 +1177,9 @@ def _info_oid(self, oid, path=None) -> Optional[OneDriveInfo]:
                 try:
                     item = self._get_item(client, oid=oid).get()
                 except OneDriveError as e:
-                    log.info("info failure %s / %s", e, e.code)
+                    log.error("info failure %s / %s", e, e.code)
```

Comment:
this isn't necessarily an error -- since we call get_latest() (which ultimately calls this func) on all changes, including deletes, we'll get a lot of these